_kubernetes/02-create_cluster.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Create Cluster
+order: 2
+duration: 2
+---
+
+## Cluster templates
+
+In Magnum, a COE Cluster is created via a Cluster Template. A Cluster Template
+allows you to specify options for setting up the Clusters. Some of the options
+include:
+
+- Availability Zone
+- Kubernetes version
+- Nova Flavor size for the nodes in the cluster
+- Glance Image for the nodes
+- Number of nodes
+
+As the amount of options can be overwhelming, Nectar provides a few default
+templates to get you started quickly. These templates are in the format
+`kubernetes-<az>-version`, and they can be viewed by doing:
+
+```
+$ openstack coe cluster template list
++--------------------------------------+------------------------------+
+| uuid                                 | name                         |
++--------------------------------------+------------------------------+
+| f561d9ee-0893-46ec-b723-a918d3ca58ce | kubernetes-melbourne-v1.14.6 |
+| bce4a43f-b5d3-467d-ab5a-820270ec0c32 | kubernetes-tasmania-v1.14.6  |
+| a8f01986-cda5-494b-8f10-fe7124340c2f | kubernetes-auckland-v1.14.6  |
+| 4f4e6a2c-b024-4bd5-9e32-6689718cf55f | kubernetes-QRIScloud-v1.14.6 |
++--------------------------------------+------------------------------+
+```
+
+You can also create your own templates for custom options. Use `$ openstack coe
+cluster template show kubernetes-melbourne-v1.14.6` to see options for the
+template, and make sure you copy all the values accordingly.
+
+Note that options `cloud_provider_tag` and `kube_tag` are highly dependent on
+the current version of the COE driver in Magnum, and the latest versions might
+not be supported in Magnum yet.
+
+## Create a Cluster
+
+Using the python command line client
+
+1. Choose a Cluster Template to create your Cluster from.
+
+1. Create the Cluster
+
+    ```
+    $ openstack coe cluster create --cluster-template kubernetes-melbourne-v1.14.6 \
+    --keypair jake mycluster
+    Request to create cluster 84091b19-fd32-4197-a92e-71838d0502c2 accepted
+    ```
+
+1. List clusters in your project. Wait for your cluster to reach the
+   `CREATE_COMPLETE` status. This may take up to 15 minutes.
```

Comment:
this took 53 minutes for me:
```
justb@DOM-IMA-13ND0N2:~$ openstack coe cluster show c5316067-7c40-4056-95f2-131b1fac463c
...
| created_at           | 2020-01-13T00:14:00+00:00
| updated_at           | 2020-01-13T01:07:40+00:00
...
```

Reply:
The cloud wasn't too stable last few days. Should be better now.